pkg/telemetry/phonehome/utils.go

```diff
@@ -0,0 +1,23 @@
+package phonehome
```

Comment:
I think "utils" files are best avoided as they are too generic. Maybe you can move this function somewhere else and delete this file all together?